components/kyma-environment-broker/common/runtime/client_test.go

```diff
@@ -59,6 +59,7 @@ func TestClient_ListRuntimes(t *testing.T) {
 			assert.ElementsMatch(t, params.RuntimeIDs, query[RuntimeIDParam])
 			assert.ElementsMatch(t, params.Regions, query[RegionParam])
 			assert.ElementsMatch(t, params.Shoots, query[ShootParam])
+			assert.ElementsMatch(t, params.Plans, query[PlanParam])
```

Comment:
You could add some plans above in `params` to test plan parameter passing of the client.

Reply:
Added and tested, parsing works well